### PR TITLE
Using the same key for storing and retrieving audit data in EventRepositoryRedis

### LINKED ISSUE #399: EventRepositoryRedis uses different keys for storing and retrieving audit data
Hi,

i am trying to use the `EventRespositoryRedis` to store monitoring data. Unfortunately the web ui always stays empty. Switching to a `JdbcEventRepository` immediately solves the problem, so i guess my basic setup is okay. Here is a snippet of my ff4j configuration:
```
...
        val ff4j = new FF4j();
        ff4j.setFeatureStore(new JdbcFeatureStore(ds));
        ff4j.setPropertiesStore(new JdbcPropertyStore(ds));
        val redisConnection = new RedisConnection(redisHost, redisPort);
        ff4j.setEventRepository(new EventRepositoryRedis(redisConnection));
        ff4j.audit(true);
...
```
After digging into the code i can see, that for storing the data a key like this is used `"FF4J_EVENT_AUDITRAIL_20200113"` while for retrieving the data a key like this is used `"FF4J_EVENT_20200113".` Only the first key exists in my redis instance. 

Is there any magic behind this or is this a bug?

### PR DESCRIPTION
**Pull request by ggerbaud**
The string action parameter in method getHashKey doesn't seem to be useful, only its nullity is checked.
It is also buggy, see #399

Fix #399

### CODE REVIEW

**Comment by clun** [comment]
Thank you so much. I am back in the game and this was on my plate for a while !

File: ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java [hunk 1]
```diff
@@ -153,13 +152,9 @@ public boolean saveEvent(Event evt) {
...
 
-    private String getHashKey(long timestamp, String action) {
-        String hashId = KEY_EVENT;
-        if (action != null) {
-            hashId += RedisContants.KEY_EVENT_AUDIT + "_";
-        }
-        long timeStamp = timestamp;
-        hashId += SDF_KEY.format(new Date(timeStamp));
+    private String getHashKey(long timestamp) {
+        String hashId = KEY_EVENT + RedisContants.KEY_EVENT_AUDIT + "_";
```
Comment (Halayem) [inline ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java:156]: return `KEY_EVENT + RedisContants.KEY_EVENT_AUDIT +  "_" + SDF_KEY.format( new Date( timestamp ) );`

File: ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java [hunk 2]
```diff
@@ -271,7 +266,7 @@ public EventSeries getAuditTrail(EventQueryDefinition query) {
         EventSeries eventSeries = new EventSeries();
         try {
             jedis = getJedis();
-            String hashKey = getHashKey(query.getFrom(), EventConstants.ACTION_CHECK_OK);
+            String hashKey = getHashKey(query.getFrom());
             Set<String> events = jedis.zrangeByScore(hashKey, query.getFrom(), query.getTo(), 0, 100);
```
Comment (Halayem) [inline ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java:270]: 4th and 5th parameters should be set in a constant and not hard coded

File: ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java [hunk 3]
```diff
@@ -139,7 +138,7 @@ public boolean saveEvent(Event evt) {
         try {
             jedis = getJedis();
             long timeStamp = evt.getTimestamp();
-            String hashId = this.getHashKey(evt.getTimestamp(), evt.getAction());
+            String hashId = this.getHashKey(evt.getTimestamp());
             evt.setUuid(String.valueOf(timeStamp));
             jedis.zadd(hashId, timeStamp, objectMapper.writeValueAsString(evt));
             return true;
```
Comment (Halayem) [inline ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java:144]: why ?

File: ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java (hunk 2 above)
Comment (ggerbaud) [reply ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java:270]: That's right, but that was not the goal of this PR

File: ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java (hunk 3 above)
Comment (ggerbaud) [reply ff4j-store-redis/src/main/java/org/ff4j/store/EventRepositoryRedis.java:144]: Because it was not really used in the hashKey

**Pull request merged**